en/css/README.md

```diff
@@ -182,12 +181,14 @@ Great!
 
 As mentioned above, CSS has a concept of classes. These allow you to name a part of the HTML code and apply styles only to this part, without affecting other parts. This can be super helpful! Maybe you have two divs that are doing something different (like your header and your post).  A class can help you make them look different.
 
-Go ahead and name some parts of the HTML code. Add a class called `page-header` to your `div` that contains your header, like this:
+Go ahead and name some parts of the HTML code. Replace the `div` that contains your header with the following:
 
 {% filename %}blog/templates/blog/post_list.html{% endfilename %}
 ```html
 <div class="page-header">
-    <h1><a href="/">Django Girls Blog</a></h1>
+    <div class="container">
+        <h1><a href="/">Django Girls Blog</a></h1>
+    </div>
 </div>
 ```
```

Comment:
Do we need the nested `div` element, or would one div with two classes work, too?
```html
<div class="page-header container">
    <h1><a href="/">Django Girls Blog</a></h1>
</div>
```

That'd make it easier to talk about in the Django Forms chapter.

Reply:
In the css definitions, `page-header` class gives the site wide orange background color for header. `container` class gives the max width property. When we put both together, it looks as follows.

![image](https://user-images.githubusercontent.com/5616249/100517001-ac045380-31ad-11eb-9d18-7064162e2cb8.png)

Reply:
@das-g @ekohl I just had an alternative thought. Lets use semantic tag in templates.

https://developer.mozilla.org/en-US/docs/Glossary/Semantics

This is the recommend way of HTML5 after all. This will make our job of referring tags easier.

#### `base.html`
```jinja
{% load static %}
<!DOCTYPE html>
<html>
    <head>
        <title>Django Girls blog</title>
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/css/bootstrap.min.css" integrity="sha384-TX8t27EcRE3e/ihU7zmQxVncDAy5uIKz4rEkgIXeMed4M0jlfIDPvg6uqKI2xXr2" crossorigin="anonymous">
        <link href="//fonts.googleapis.com/css?family=Lobster&subset=latin,latin-ext" rel="stylesheet" type="text/css">
        <link rel="stylesheet" href="{% static 'css/blog.css' %}">
    </head>
    <body>
        <header class="page-header">
            <div class="container">
            {% if user.is_authenticated %}
                <a href="{% url 'post_new' %}" class="top-menu">
                    {% include './icons/file-earmark-plus.svg' %}
                </a>
            {% endif %}
                <h1><a href="/">Django Girls Blog</a></h1>
            </div>
        </header>
        <main class="container">
            <div class="row">
                <div class="col">
                {% block content %}
                {% endblock %}
                </div>
            </div>
        </main>
    </body>
</html>
```

#### `post_detail.html`
```jinja
{% extends 'blog/base.html' %}

{% block content %}
    <article class="post">
    	<aside class="actions">
        {% if post.published_date %}
            <time class="date">
                {{ post.published_date }}
            </time>
        {% endif %}
        {% if user.is_authenticated %}
             <a class="btn btn-default btn-sm" href="{% url 'post_edit' pk=post.pk %}">
                {% include './icons/pencil-fill.svg' %}
             </a>
        {% endif %}
		</aside>
        <h2>{{ post.title }}</h2>
        <p>{{ post.text|linebreaksbr }}</p>
    </article>
{% endblock %}
```

#### `post_list.html`
```jinja
{% extends 'blog/base.html' %}

{% block content %}
    {% for post in posts %}
        <article class="post">
            <time class="date">
                {{ post.published_date }}
            </time>
            <h2><a href="{% url 'post_detail' pk=post.pk %}">{{ post.title }}</a></h2>
            <p>{{ post.text|linebreaksbr|truncatechars:77 }}</p>
        </article>
    {% endfor %}
{% endblock %}
 ```

#### `blog.css`
```css
.page-header {
    background-color: #C25100;
    margin-top: 0;
    margin-bottom: 40px;
    padding: 20px 20px 20px 40px;
}

.page-header h1,
.page-header h1 a,
.page-header h1 a:visited,
.page-header h1 a:active {
    color: #ffffff;
    font-size: 36pt;
    text-decoration: none;
}

h1,
h2,
h3,
h4 {
    font-family: 'Lobster', cursive;
}

.date {
    color: #828282;
}

.save {
    float: right;
}

.post-form textarea,
.post-form input {
    width: 100%;
}

.top-menu,
.top-menu:hover,
.top-menu:visited {
    color: #ffffff;
    float: right;
    font-size: 26pt;
    margin-right: 20px;
}

.post {
    margin-bottom: 70px;
}

.post h2 a,
.post h2 a:visited {
    color: #000000;
}

.post > .date,
.post > .actions {
    float: right;
}


.btn-default,
.btn-default:visited {
    color: #C25100;
    background: none;
    border-color: #C25100;
}

.btn-default:hover {
    color: #FFFFFF;
    background-color: #C25100;
}
```

Reply:
Semantic elements indeed sound like a good idea. If I understand correctly, the simplification would be due to not fewer elements in total, but fewer of the same type at the same place, therefore making it easier to convey what element or tag we're referring to.

Reply:
@das-g Exactly. Same code with unqiuely identifiable tags. 

Btw, I am not sure if `<aside>` tag is used in correct context there. Never used that tag in my life. xD Maybe we should simple use `div` tag instead of `aside` tag. Same goes to `time` tag. Never used in real senario before. But I think it probably helps search engine to show new contents from the site on searchs based on these tags.

I have some research on new SEO techniques and found that following semantics tags and HTML5 structure is huge jump up the search ranking.

Reply:
Looking at [the HTML5 spec for `aside`](https://www.w3.org/TR/html52/sections.html#the-aside-element) (especially [example 15](example-650ac100)) and [MDN's documention for `time`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time), I think these are semantically correct uses of these two elements. :+1:

Reply:
I like that solution.

Reply:
@ekohl @das-g Please review https://github.com/DjangoGirls/tutorial/pull/1681/commits/a9dbbf582e5e863bbb4ef50a16201a2b758a81f9